### ISSUE TITLE
Accessibility issue, a chessgame cannot be played with a screen reader.

### ISSUE THREAD

**Issue by rahulbhai9** [lichess-org/mobile#869]
As of now the chess board cannot be navigated with the screen reader (Talkback). I cannot focus each square individually with the screen reader. It means I cannot click on a specific square for making a move. As of now no visually impaired person can play chess using the app because of this bug. Ideally each square of the chess board must be individually focusable with the screen reader. And visually impaird person should be able to make move by clicking on source square and the target square. Thanks.

**Comment by ZTL-UwU** [comment]
Duplicated with #28
